src/utils/Lookups.ts

```diff
@@ -47,13 +49,35 @@ export function productById(id: string, products: Product[]): Product {
 }
 
 export function renderDateTime(epoch: number) {
-    // Todo: locale should be configurable for GPL
-    return isEmpty(epoch) ? "" : new Date(epoch * 1000).toLocaleString("nl-NL");
+    // Convert timestamp to ISO8601-like format.
+    // Example:
+    //   2023-07-12 15:16:32 CEST
+    if (!epoch) {
+        return "";
+    }
+    // get milliseconds for unix timestamp
+    const time = moment(epoch * 1000);
+    // get timezone from translations object
+    const timezone = intl.formatMessage({ id: "locale.timezone" });
+    const format = "y-MM-DD HH:mm:ss z";
+
+    return time.tz(timezone).format(format);
```

Comment:
You are right that the format will likely be different. If that's not ideal then should ISO8601 not be the default? I can try to make the format string configurable...

Having a try/catch with the user's browser locale string is a good fallback. I will add it.

The current `src/locale/en.ts` file does currently contain "Europe/Amsterdam", in case the user does not set explicitly or override with backend translations file.

```javascript
{ ...
    locale: {
        timezone: "Europe/Amsterdam",
    },
};

```